File: Computadores/Computadores/src/main/java/com/example/Computadores/model/Computador.java
package com.example.Computadores.model;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;
import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Computador {
    private int codigo;
    private String marca;
    @JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss")
    private LocalDateTime fechaFabricacion;
    private String estado;
    private boolean portatil;
    private Double costoMantenimiento;
    private boolean activo = true;
    private List<String> perifericos;
}

File: ClienteComputador/ClienteComputador/ClienteComputador/Computador.cs
using System.Collections.Generic;

namespace ClienteComputador
{
    public class Computador
    {
        public int codigo { get; set; }
        public string marca { get; set; } = "";
        public string fechaFabricacion { get; set; } = "";
        public string estado { get; set; } = "";
        public bool portatil { get; set; }
        public double costoMantenimiento { get; set; }
        public bool activo { get; set; }
        public List<string>? perifericos { get; set; }

        // Para mostrar en la grilla
        public string PortatilStr => portatil ? "Si" : "No";
        public string ActivoStr => activo ? "Activo" : "Inactivo";
        public string PericosStr => perifericos != null ? string.Join(", ", perifericos) : "";
    }
}
